py_mmd_tools/nc_to_mmd.py

```diff
@@ -1264,49 +1269,34 @@ def to_mmd(self, collection=None, checksum_calculation=False, mmd_yaml=None,
 
         # Data access should not be read from the netCDF-CF file
         mmd_yaml.pop('data_access')
-        # Add OPeNDAP data_access if "netcdf_product" is OPeNDAP url
-        rm_file_for_checksum_calculation = False
-        if 'dodsC' in self.netcdf_product:
+        # Add OPeNDAP data_access if opendap_url is not None
+        if self.opendap_url is not None:
             self.metadata['data_access'] = self.get_data_access_dict(ncin, **kwargs)
-            resource = ''
-            for access in self.metadata['data_access']:
-                if access['type'] == 'HTTP':
-                    resource = access['resource']
-            logging.info('Downloading NetCDF file to calculate checksum...')
-            file_for_checksum_calculation = wget.download(resource)
-            rm_file_for_checksum_calculation = True
         else:
             self.metadata['data_access'] = []
-            file_for_checksum_calculation = self.netcdf_product
 
-        file_size = pathlib.Path(file_for_checksum_calculation).stat().st_size / (1024 * 1024)
+        file_size = pathlib.Path(self.netcdf_file).stat().st_size / (1024 * 1024)
 
         for key in mmd_yaml:
             self.metadata[key] = self.get_acdd_metadata(mmd_yaml[key], ncin, key)
 
         # Set storage_information
-
-        file_location = self.netcdf_product
-
         self.metadata['storage_information'] = {
-            'file_name': os.path.basename(self.netcdf_product),
-            'file_location': file_location,
+            'file_name': os.path.basename(self.netcdf_file),
+            'file_location': os.path.dirname(self.netcdf_file),
```

Comment:
Only works if full path is given.
```
>>> os.path.dirname("hello.txt")
''
>>> os.path.dirname("~/hello.txt")
'~'
>>> os.path.dirname("home/user/hello.txt")
'home/user'
```

Reply:
hmm - må vi legge inn mer da?

Reply:
kunne sjekket at det er en faktisk sti som begynner med "/" og ikke inneheolder "~". Eller kanskje det finnes en funksjon som kan brukes allerede?

Reply:
uansett, så er det noe ansvar som må legges på MMD-produsenten..

Reply:
ja, vi bør kreve full path

Reply:
løst i bb55c5c02fc28cc209a9a954095176bb4da241e0, men må lage test...